### ISSUE TITLE
Create MVP App Design

### ISSUE THREAD

**Issue by ugur-eren** [keep-starknet-strange/joyboy#100]
### OD Hack

This issue is part of ODHack 4.0:

description - https://x.com/OnlyDust_com/status/1791035111992934563
guidelines - https://onlydust.notion.site/ODHack-Common-Guidelines-Project-leads-5e21836f85ad493f9ca429770af0ad40

Please read the guidelines and make sure you follow them.

**Please write your thoughts and how long you think it will take to complete the issue in the comments.**

### Create MVP App Design

We need to have a design ready for the MVP version of the app. The design should be simple and should be able to be implemented in a short time.

Required screens for the design are:
- Auth flow, Login and Create Account
- Feed
- Create Post
- Profile (both user and other users)
- Edit Profile
- Post Comments

And any other screens you think are necessary.

@Israelrex9 If you can comment I can assign this to you.

### Communication

https://t.me/JoyboyStarknet/949

May the Joy be with you!

**Comment by bhavyagosai** [comment]
Hey

**Comment by Israelrex9** [comment]
yeah, I am working on MVP and will see it to the end. Thanks @ugur-eren

**Comment by ugur-eren** [comment]
Thanks for the great work!
